doc/rst/usingchapel/building.rst

```diff
@@ -62,132 +56,156 @@ display its help message using:
+  standard technique is still required to ensure that ``chpl`` is in
+  your path.
+
+* by installing Chapel to a specific location—perhaps one that is
+  already in your path—using the instructions in the next section.
+
+
+.. _readme-installing:
+
+-----------------
+Installing Chapel
+-----------------
+
+Chapel can be built and installed to a specific location as follows:
+
+.. code-block:: bash
+
+  ./configure  # use './configure --help' for specific options
+  make
+  make install
+
+Running ``./configure`` will save your current configuration of
+``$CHPL_`` settings into a ``chplconfig`` file.  Running it with the
+``--prefix`` or ``--chpl-home`` options permits you to specify where
+and how Chapel should be installed during the ``make install`` step.
+Specifically:
+
+* ``--prefix=/dir/for/install/`` causes the Chapel compiler,
+  libraries, and supporting code to be installed into the directories:
+
+  - ``/dir/for/install/bin``
+  - ``/dir/for/install/lib``
+  - ``/dir/for/install/share``
+
+  This technique is designed to install Chapel using a standard
+  directory structure for the purposes of integrating it into a
+  standard location that is already in your path, such as
+  ``/usr/local/`` or ``~/``.  Note that elevated privileges are likely
+  to be required for any system-wide installation locations.
+
+* ``--chpl-home=/dir/for/install`` copies key files and directories
+  from the Chapel source tree into ``/dir/for/install``, preserving
+  Chapel's traditional directory structure.
+
+
+-----------------------------------------
+Switching Between Multiple Configurations
+-----------------------------------------
+
+A single installation of Chapel can simultaneously support multiple
+configuration options, such as target platforms or compilers, because
+all binary files and executables are stored in unique subdirectories
+as determined by the ``CHPL_`` environment variables.  However, note
+that each ``make`` command only builds the compiler and runtime for
+the current set of ``CHPL_`` environment variables defined by—and
+inferred for—your environment.  To build support for additional
+configurations, you will need to modify your ``CHPL_`` environment
+variables and re-make.
+
+Once you have built multiple configurations, you can switch between
+them either by setting your ``CHPL_`` environment variables to the
+same values as when the configuration was built, or using the ``chpl``
+compiler's command-line options that correspond to the environment
+variables (see the 'Compiler Configuration Options' section of the
+:ref:`man-chpl` man page for details).
 
 
 ----------------
 Makefile Targets
 ----------------
 
-The Chapel sources are structured so that a GNU-compatible make
+The Chapel sources are structured so that a GNU-compatible ``make``
 utility can be used in any source directory to build the sources
 contained in that directory and its subdirectories.  All of these
```

Comment:
It's not the case that every Makefile in the project supports all of these projects. In particular, as far as I know, `make check` only works on the toplevel Makefile.

Not sure what the right way to fix that is.

Reply:
I'm going to say "not it!" on this one since it's a pre-existing condition without a clear fix and not part of what I set out to tackle in this edit.  If we wanted to pin it on someone, we could turn to @e-kayrakli as reviewer for #17442, but I think it's close enough to the truth that given the very small number of people who probably read this section that it's probably not a big problem.  Certainly not something that gives me any pause.